src/mprk.jl

```diff
@@ -281,6 +258,8 @@ function perform_step!(integrator, cache::MPEConstantCache, repeat_step = false)
 
     # Attention: Implementation assumes that the pds is conservative,
     # i.e., P[i, i] == 0 for all i
+    # We use P to store the last evaluation of the PDS 
+    # as well as to store the system matrix of the linear system    
```

Comment:
It's still `M` below in the out-of-place version here. Should this comment be moved to the in-place version?

Reply:
Absolutely! I hope the comments are now where they belong.